### PR TITLE
added AskForPermissionConsent card as well as test case

### PR DESCRIPTION
**Pull request by chrischang**
I wanted to check the consent exists in every request, so I wrote the example is as following:

```
app.pre = function(request, response, type) {
  if (!request.context.System.user.permissions || !request.context.System.user.permissions.consentToken) {
    response.say("please enable your location. I have sent the permission card to your companion app. Just go to alexa.amazon.com to enable it.");
    response.card({
      type: 'AskForPermissionsConsent',
      permissions: [ 'read::alexa:device:all:address:country_and_postal_code' ]
    });
    response.fail('location is required');
  }
}
```
Hope this helps~ 
Thank you :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ### 4.0.1 (Next)
 
+* [#223](https://github.com/alexa-js/alexa-app/issues/223): new card type available, AskForPermissionsConsent - [@ericblade]
```
Comment (dblock) [inline CHANGELOG.md]: Lets make this consistent with the other changelog lines, capital first letter, a period in the end and a link to the github profile. Maybe:

```
* [#223](https://github.com/alexa-js/alexa-app/issues/223): Added support for `AskForPermissionsConsent` cards - [@ericblade](https://github.com/ericblade).
```

**Comment by dblock** [comment]
This needs to be documented in README, too, please.

**Comment by chrischang** [comment]
Thanks for the feedback @dblock. Will add them tonight :)

**Comment by ericblade** [comment]
FWIW this would fix #223

**Comment by chrischang** [comment]
OK! Let me know if there's anything that needs to be changed.

**Comment by dblock** [comment]
Thanks, merged.

**Pull request merged**